src/status_im2/contexts/shell/components/floating_screens/view.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im2.contexts.shell.components.floating-screens.view
+  (:require [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [react-native.reanimated :as reanimated]
+            [status-im2.contexts.shell.state :as state]
+            [status-im2.contexts.chat.messages.view :as chat]
+            [status-im2.contexts.shell.animation :as animation]
+            [status-im2.contexts.shell.constants :as shell.constants]
+            [status-im2.contexts.shell.components.floating-screens.style :as style]
+            [status-im2.contexts.communities.overview.view :as communities.overview]))
+
+(def screens-map
+  {shell.constants/community-screen communities.overview/overview
+   shell.constants/chat-screen      chat/chat})
+
+(defn f-screen
+  [screen-id {:keys [id animation] :as screen-param}]
+  ;; First render screen, then animate (smoother animation)
+  (rn/use-effect
+   (fn []
+     (animation/animate-floating-screen screen-id screen-param))
```

Comment:
The timer returned by the call is not being cleared on unmount.

Reply:
https://github.com/status-im/status-mobile/pull/15893#discussion_r1212130919